File: php/utility/MakeContext.php
<?php
declare(strict_types=1);

// NeugeborenenVornamenKantonStgallen SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class NeugeborenenVornamenKantonStgallenMakeContext
{
    public static function call(array $ctxmap, ?NeugeborenenVornamenKantonStgallenContext $basectx): NeugeborenenVornamenKantonStgallenContext
    {
        return new NeugeborenenVornamenKantonStgallenContext($ctxmap, $basectx);
    }
}

File: .sdk/tm/rb/utility/make_context.rb
# NeugeborenenVornamenKantonStgallen SDK utility: make_context
require_relative '../core/context'
module NeugeborenenVornamenKantonStgallenUtilities
  MakeContext = ->(ctxmap, basectx) {
    NeugeborenenVornamenKantonStgallenContext.new(ctxmap, basectx)
  }
end
